### PR TITLE
Add support for Xenial with install-globaleaks.sh

### PR DESCRIPTION
**Pull request by NSkelsey**
This PR tracks the changes to the build/packaging and related scripts to support Xenial 16.04.

There are some large changes in here:

0) The appliance changes here should produce a functional Xenial image using the instructions found in `appliance/Readme.md`

1) The Xenial package uses the dependencies in debian/controlX/control.xenial as the definition of its dependencies.

2) The requirements check for xenial requirements is performed with a `>=` instead of a hard equals at bootup. This allows the deps to shift underneath according to the updates of the system.

3) The apparmor profile was slightly expanded. `usr.bin.globaleaks` was asking to use `/proc/` with xenial so I added the read permission.

Some notes:

The entire process from the build to installation has been tested in local vms for both Xenial and Trusty, they both work start to finish. 

If the behavior of the trusty build changes due to changes introduced here it is a bug.

Related to #1809

### CODE REVIEW

File: debian/controlX/control.xenial [hunk 1]
```diff
@@ -20,22 +20,21 @@ Depends:
...
+ , python-cryptography (>=1.2.3)
+ , python-cyclone (>=1.1)
+ , python-gnupg (>=0.3.8)
+ , python-openssl (>=0.15.1)
+ , python-scrypt (>=0.6.1)
+ , python-storm (>=0.19)
+ , python-transaction (>=1.4.3)
+ , python-twisted-core (>=16.0.0-1)
+ , python-txsocksx (>=1.15.0.2-1)
+ , ${python:Depends}
```
Comment (NSkelsey) [inline debian/controlX/control.xenial]: @evilaliv3 `{python:Depends}` has snuck in again. The deb package as is installs without a hitch though.

**Comment by NSkelsey** [comment]
@evilaliv3 this just needs a relabel to be integrated into devel

**Comment by NSkelsey** [comment]
Will be merging this on 17/12

**Comment by evilaliv3** [comment]
okay!

**Pull request merged**